yb_migrate/src/migration/postgres.go

```diff
@@ -338,6 +339,15 @@ func getMappingForTableNameVsTableFileName(dataDirPath string) map[string]string
 	return tableNameVsFileNameMap
 }
 
+func nameContainsCapitalLetter(name string) bool {
```

Comment:
same question here, if table as double quotes is this function then that needs to taken care.
consider using `unicode.IsLetter()` in that case

Reply:
Refer answer to the previous comment.